mailpoet/assets/js/src/homepage/components/resources.tsx

```diff
@@ -0,0 +1,13 @@
+import { MailPoet } from 'mailpoet';
+import { ContentSection } from './content-section';
+
+export function Resources(): JSX.Element {
```

Comment:
@samnajian Can you please explain why it is better to leave it on TS?

I'm asking because I see that we use this return type in almost all new components. Personally, I think that it is a good idea to explicitly define return types for simple functions where the type plays a role as a guard so that, for example, a function that is expected to return a number wouldn't suddenly start returning null. But I'm not sure about the components.

Reply:
I'd agree 100% with normal functions/methods, so that we don't need to check the function to know what is returns, but for components, they are well, components! the return type is clear us and TS, and adding explicit types is simply adding code for no special benefit, they are redundant, and besides TS is trying hard to be more smart in inferring types and nag while building ( and we watch the code with TS ), so leaving the job to TS type inference is safe and reduces the number of characters we end up writing, still please note all of these is simply a suggestion and explicitly adding return types doesn't have any known negative effects.

Reply:
Thank you! I understand. So I choose to keep those types to be consistent with most of our codebase.